pypi.sh

```diff
@@ -5,11 +5,12 @@
 # 3. Update README with new features/functions/tutorials
 # 4. Determine minimum versions and dependencies with ./min.py
 # 5. Bump minimum versions and dependencies
-#    a) setup.py
+#    a) pyproject.toml
 #    b) requirements.txt
 #    c) environment.yml
 #    d) .github/worflows/github-actions.yml
 #    e) recipes/meta.yaml in conda-feedstock
+#    f) README.md
```

Comment:
So, the latest version of `pypi.sh` already contains the updated lines:

https://github.com/TDAmeritrade/stumpy/blob/887feb8b8b6800e2e09d8f00ea2023caadad0427/pypi.sh#L8-L13

It is rather curious that your code base is now adding this line back in. This leads me to believe that your current development branch does not contain the latest updates.